sw/device/lib/testing/aon_timer_testutils.c

```diff
@@ -0,0 +1,35 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/aon_timer_testutils.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/dif/dif_aon_timer.h"
+#include "sw/device/lib/testing/check.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
+void aon_timer_wakeup_config(uint32_t cycles) {
```

Comment:
I think the naming convention we are using for the function is: `<filename>_<func_name>`, so in this case, this would be: `aon_timer_testutils_wakeup_config(...)`.

See the functions in `sw/device/lib/testing/rand_testutils.c` for an example.

If the function is not part of the public *_testutils lib you are making, then you can name it whatever you want, just mark it static.

Reply:
Done